.travis.yml

```diff
@@ -39,6 +39,9 @@ jobs:
       install:
         # Clone RMG-database
         - git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
+        # install Q2DTor
+        - git clone https://github.com/mjohnson541/Q2DTor.git RMG-Py/external/Q2DTor --branch arkane
```

Comment:
perhaps use the make command you added, that way it'll test that as well

Reply:
The problem is that make command requires an ENTER command.  This is much simpler and easier to read for travis purposes.

Reply:
gotcha. and we can't do something like `make q2dtor -y`, or can we?